pkg/ui/src/views/cluster/containers/dataDistribution/replicaMatrix.tsx

```diff
@@ -94,11 +95,41 @@ class ReplicaMatrix extends Component<ReplicaMatrixProps, ReplicaMatrixState> {
     return `${arrow} ${label}`;
   }
 
+  renderCell(
+    row: FlattenedNode<SchemaObject>,
+    col: FlattenedNode<NodeDescriptor$Properties>,
+    scale: d3.scale.Linear<number, number>,
+  ) {
+    if (!(row.isLeaf || row.isCollapsed)) {
+      return null;
+    }
+
+    const value = sumValuesUnderPaths(
+      this.props.rows, this.props.cols, row.path, col.path, this.props.getValue,
+    );
+
+    if (value === 0) {
+      return null;
+    }
+
+    const lightnessValue = scale(value);
+    const backgroundColor = `hsl(210, 100%, ${lightnessValue}%)`;
```

Comment:
should probably be using this https://github.com/d3/d3-scale-chromatic#sequential-single-hue instead of my janky color interpolation here; @couchand plz advise on the appropriate d3-ism :P